.gitmodules

```diff
@@ -0,0 +1,141 @@
+[submodule "ovos-backend-client"]
+	path = ovos-backend-client
+	url = https://github.com/OpenVoiceOS/ovos-backend-client
+[submodule "ovos-core"]
+	path = ovos-core
+	url = https://github.com/OpenVoiceOS/ovos-core
+[submodule "ovos-audio"]
+	path = ovos-audio
+	url = https://github.com/OpenVoiceOS/ovos-audio
+[submodule "ovos-ocp-audio-plugin"]
+	path = ovos-ocp-audio-plugin
+	url = https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin
+[submodule "ovos-messagebus"]
+	path = ovos-messagebus
+	url = https://github.com/OpenVoiceOS/ovos-messagebus
+[submodule "ovos-dinkum-listener"]
+	path = ovos-dinkum-listener
+	url = https://github.com/OpenVoiceOS/ovos-dinkum-listener
+[submodule "ovos-vad-plugin-silero"]
+	path = ovos-vad-plugin-silero
+	url = https://github.com/OpenVoiceOS/ovos-vad-plugin-silero
+[submodule "ovos-ww-plugin-pocketsphinx"]
+	path = ovos-ww-plugin-pocketsphinx
+	url = https://github.com/OpenVoiceOS/ovos-ww-plugin-pocketsphinx
+[submodule "ovos-audio-transformer-plugin-ggwave"]
+	path = ovos-audio-transformer-plugin-ggwave
+	url = https://github.com/OpenVoiceOS/ovos-audio-transformer-plugin-ggwave
+[submodule "ovos-ww-plugin-precise"]
+	path = ovos-ww-plugin-precise
+	url = https://github.com/OpenVoiceOS/ovos-ww-plugin-precise
```

Comment:
Up for discussion, but maybe we should not include deprecated packages, or maybe we should so they are in at least 1 release, I will leave a comment in all deprecated packages for reference 

DEPRECATED: ovos-ww-plugin-precise

Reply:
the list of repos is definitely up for discussion